talking-about-their-pronouns.md

```diff
@@ -0,0 +1,38 @@
+# Asking someone's pronouns
+
+## Over Slack / in conversation
+
+What: Sorry, just to check so I can get them right, do you mind me asking what your pronouns are?
+
+# Correcting someone who misgenders someone else
+
+## During a meeting
+
+### Respond in the moment
+
+When: Once the person who's just got the pronouns incorrect has finished talking
+How: Verbally
+What: Just to point out, I'm pretty sure their pronouns are they/them
```

Comment:
```suggestion
What: [when you next refer to them, use the pronouns you know the person uses]
```

I think this is a really cool idea - it certainly reduces the cognitive load on me if someone else is able to do this for me, especially when I'm not around.

I would suggest not making a big deal of this in the moment - you're likely gathered for some other reason apart from their pronouns!

Reply:
I like to go beyond just using the right pronouns as it feels almost passive aggressive to just use the right one without explaining why? Maybe this should highlight the context dependency - sometimes I'd definitely just mention it later but sometimes it can just be a quick thing before the conversation moves on? I like doing it explicitly if I'm in a conversation where no one present knows the person's gender too to challenge assumptions of gender (most commonly from experience: they're a dev so they're male)